src/content/docs/en/guides/deploy/azion.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Deploy your Astro Site to Azion
+description: How to deploy your Astro site to the web using Azion.
+type: deploy
+i18nReady: true
+---
+import ReadMore from '~/components/ReadMore.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+You can deploy your Astro project on [Azion](https://console.azion.com/), a platform for frontend developers to collaborate and deploy static (JAMstack) and SSR websites.
+
+This guide includes:
+
+- [How to Deploy through Azion Console Dashboard](#how-to-deploy-a-static-site-github)
+- [How to deploy using the Azion CLI](#how-to-deploy-a-site-using-azion-cli)
+
```

Comment:
This portion does not seems necessary, readers can navigate through the Table of Contents and the links mentioned here aren't linked exactly with headings. Might as well remove the fluff
```suggestion
```